example/Dll.c

```diff
@@ -206,7 +206,7 @@ void onUnitMorph(AIModule* module, Unit* unit) {}
 void onUnitRenegade(AIModule* module, Unit* unit) {}
 void onSaveGame(AIModule* module, const char* gameName) {}
 void onUnitComplete(AIModule* module, Unit* unit) {}
-
+void drop(AIModule* module) {}
 static AIModule_vtable module_vtable = {
```

Comment:
I'd make an empty line below `void drop(AIModule* module) {}` to make the struct visually separated.